libretroshare/src/pqi/authssl.cc

```diff
@@ -1198,6 +1198,21 @@ int AuthSSLimpl::VerifyX509Callback(int /*preverify_ok*/, X509_STORE_CTX* ctx)
 	std::string sslCn = RsX509Cert::getCertIssuerString(*x509Cert);
 	RsPgpId pgpId(sslCn);
 
+    RsPeerDetails det;
+    if(!rsPeers->getPeerDetails(sslId,det))
```

Comment:
This extra check deny connection to a new location of a PGP friend, IMHO we should remove it